src/utils/getArticleContents.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+const { parseHTML } = require("linkedom");
+
+/*
+ * Given page content markup, this function prepares a list of links to all headings of a given level
+ * followed by the page content with ID's injected for each linked heading.
+ *
+ * This work was undertaken to address GitHub issue #59:
+ * https://github.com/fluid-project/handbook.floeproject.org/issues/59
+ *
+ * @param {String} pageContent - the page content markup
+ * @param {String} [summary] - the summary text to display at the top of the article contents list
+ * @param {String} [headingsSelector] - a CSS selector to indicate which heading elements to list
+ * @param {String} [containerCssClass] - a CSS class for the article contents container element
+ *
+ * @return {String} - the combined string containing the list markup and modified page markup
+ */
+module.exports = (pageContent, summary = "Article Contents", headingsSelector = "h2", containerCssClass = "article-contents") => {
+    const {document} = parseHTML(pageContent);
+    const headings = document.querySelectorAll(headingsSelector);
+
+    const summaryCssClass = containerCssClass + "__summary";
+    const listCssClass = containerCssClass + "__list";
+    const listItemCssClass = containerCssClass + "__list-item";
+    const linkCssClass = containerCssClass + "__link";
```

Comment:
This is fine, but I tend to find it easier to read with template literals. e.g. `${containerCssClass}__summary`.

Reply:
This is a reasonable suggestion. I'm stuck in my old C-like ways and sometimes forget the template literals are a thing :)